.github/CHANGELOG.md

```diff
@@ -2,6 +2,35 @@
 
 <h3>New features since last release</h3>
 
+* PennyLane now supports a new device, `default.mixed`, designed for
+  simulating mixed-state quantum computations. This enables native
+  support for implementing noisy channels in a circuit, which generally
+  map pure states to mixed states. The device can be initialized as
+  ```python3
+  >>> dev = qml.device("default.mixed", wires=1)
+  >>> print(dev.state)
```

Comment:
Do we want to advertise `dev.state`? I've always thought of it as a more "behind-the-scenes" sort of thing, and would be ok if this snippet just had the device definition.

Reply:
:+1: Especially if we plan on having `return qml.state()` work and return density matrices for `default.mixed`.